components/collapsible-panel/collapsible-panel.js

```diff
@@ -298,6 +287,21 @@ class CollapsiblePanel extends RtlMixin(LitElement) {
 		`;
 	}
 
+	updated(changedProperties) {
+		if (changedProperties.has('expanded')) {
+			if (!this.expanded) {
+				const element = this.shadowRoot.querySelector('.d2l-collapsible-panel');
+				element.classList.remove('scrolled');
+			}
+
+			const event = `d2l-collapsible-panel-${this.expanded ? 'expand' : 'collapse' }`;
+
+			this.dispatchEvent(new CustomEvent(
```

Comment:
Typically we only want to fire events based on user interaction (i.e. the user clicking to expand or collapse), not based on programatic changes. The way this is currently, `elem.expanded = false;` will fire the event. Probably want to move this back into `_toggleExpand()`?

Reply:
Oh sorry, just re-read [your comment](https://github.com/BrightspaceUI/core/pull/3025#discussion_r1053643225) which I initially misread...will fix.

> You likely don't want to fire the event though from there, so that part could stay here.